package.json

```diff
@@ -28,6 +28,7 @@
     "backoff": "2.5.0",
     "body-parser": "1.15.1",
     "datascript": "0.15.0",
+    "datascript-mori": "0.16.0",
```

Comment:
I think you only need one of these, since `datascript-mori` provides both.  Drop the former.